darts/models/forecasting_model.py

```diff
@@ -154,6 +154,7 @@ def historical_forecasts(self,
                              forecast_horizon: int = 1,
                              stride: int = 1,
                              retrain: bool = True,
+                             retrain_epochs: int = 100,
```

Comment:
@hrzn this `retrain_epochs` param is only used when `TorchForecastingModel` is being used. The default 100 epochs was a previous default for `n_epochs`.

Reply:
Also: do you want UTs for new params? I think they should be.